engine/classes/ElggPlugin.php

```diff
@@ -140,6 +140,19 @@ public function getFriendlyName() {
 	}
 
 	/**
+	 * Returns the manifest's name if available, otherwise the ID.
+	 *
+	 * @return string
+	 * @since 1.8.1
+	 * @deprecated 3.0
+	 */
+	public function getFriendlyName() {
+		elgg_deprecated_notice('ElggPlugin->getFriendlyName() is deprecated. Use getDisplayName().', '3.0');
```

Comment:
changed this to `elgg_deprecated_notice(__METHOD__ . ' is deprecated. Use getDisplayName().', '3.0');` like in other places in core